pkg/oc/cli/admin/diagnostics/diagnostics/cluster/app_create/app.go

```diff
@@ -124,3 +128,44 @@ There are many reasons why this can occur; for example:
 	`, now(), d.deployTimeout))
 	return false
 }
+
+// podContainerRunning returns false until the named container has ContainerStatus running (at least once),
+// and will return an error if the pod is deleted, runs to completion, or the container pod is not available.
+func podContainerRunning(containerName string) watch.ConditionFunc {
```

Comment:
@deads2k this is a copy of the helper under oc/lib/conditions. This diagnostics file deals with internalversions, and we'll be removing diagnostics soon anyway, so I figured we could leave this copy here until then